src/main/resources/recipes.json

```diff
@@ -415,4 +415,31 @@
 		],
 		"emoji": "\ud83c\udf6a"
 	}
-]
```

Comment:
one thing needs to be fixed, each recipe should be wrapped in semi colons {}, and make sure on line 417 you add a comma after that recipe